website/app/components/doc-landing-callout-blocks.hbs

```diff
@@ -0,0 +1,7 @@
+<ul class="doc-landing-callout-blocks doc-landing-callout-blocks--layout-{{@cols}}cols">
+  {{#each @items as |item|}}
+  <li class="doc-landing-callout-blocks__item">
```

Comment:
[very minor nit that I'm not sure why the linter doesn't complain] this block should be indented inside the `{{#each}}`